app/bundles/EmailBundle/EventListener/CampaignSubscriber.php

```diff
@@ -96,7 +96,10 @@ public function onCampaignBuild(CampaignBuilderEvent $event)
     public function onEmailOpen(EmailOpenEvent $event)
     {
         $email = $event->getEmail();
-        $this->factory->getModel('campaign.event')->triggerEvent('email.open', $email, 'email.open' . $email->getId());
+        if($email != null)
```

Comment:
I know this is minor, but might it look like

```
if ($email !== null) {
```
instead to keep the code standards?

If `($email != null)` is enough, maybe `(!email)` would do the same job.